context/app/static/js/shared-styles/sections/Description/Description.jsx

```diff
@@ -7,7 +7,9 @@ function Description({ padding, children, ...props }) {
   return (
     <StyledPaper $padding={padding} {...props}>
       <StyledInfoIcon color="primary" />
-      <Typography variant="body1">{children}</Typography>
+      <Typography variant="body1" as="div">
```

Comment:
```suggestion
      <Typography variant="body1" component="div">
```
Is `as` a Typography prop or html attribute I'm not familiar with?

Reply:
I think I mistakenly used `as` because I'm more used to that prop name than `component`. It doesn't appear to be documented anywhere, but it did work as expected (i.e. it turned the container for the description into a div, so we didn't have nested `p > p` or `p > div` errors)

I'll swap this over to `component` though, I have no idea how this was functional.